src/common/paths/query_ast.yml

```
@@ -13,20 +38,58 @@ post:
         enum:
           - application/json
   requestBody:
-    description: Analyzed Flux query to generate abstract syntax tree.
+    description: Analyzed Flux query to generate an AST.
     content:
       application/json:
         schema:
           $ref: "../schemas/LanguageRequest.yml"
+  x-codeSamples:
+    - lang: Shell
+      label: cURL
+      source: |
+        curl --request POST 'INFLUX_URL/api/v2/query/ast' \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json \
+        --header 'Authorization: Token INFLUX_TOKEN' \
+        --data 'from(bucket: "example-bucket")
+                  |> range(start: -5m)
+                  |> filter(fn: (r) => r._measurement == "example-measurement")'
   responses:
     "200":
-      description: Abstract syntax tree of the flux query.
+      description: |
+        Successful request.
+        AST of the Flux query.
```

```suggestion
      description: |
        Success.
        The response body contains an Abstract Syntax Tree (AST) of the Flux query.
```

This is the pattern I've been using for everything - mostly follows Google Developer Style.